pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -106,6 +107,7 @@ public AbstractApexRule() {
         definePropertyDescriptor(CODECLIMATE_CATEGORIES);
         definePropertyDescriptor(CODECLIMATE_REMEDIATION_MULTIPLIER);
         definePropertyDescriptor(CODECLIMATE_BLOCK_HIGHLIGHTING);
+        definePropertyDescriptor(new IntegerProperty("codacy_minutes_to_fix", "Codacy.com Time to fix in minutes", Integer.MIN_VALUE, Integer.MAX_VALUE, 5, 1.0f));
```

Comment:
@hooperbloob this isn't codeclimate related, but a different tool for which there is no rendered within PMD itself. That's why I agree with https://github.com/pmd/pmd/pull/329#issuecomment-294616955 to just move these away from the PMD's main source.